viz_scripts/generic_metrics.ipynb

```diff
@@ -304,14 +471,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plot_title_no_quality=\"Average \"+ label_units+\" for each mode with > 3 entries\"\n",
     "file_name ='average_miles_mode_confirm%s' % file_suffix\n",
+    "plot_title_no_quality=\"Average \"+ label_units+\" for each mode with > 3 entries\"\n",
     "\n",
     "try:\n",
+    "    dist = expanded_ct.groupby('Mode_confirm').agg({distance_col: ['sum', 'count' , 'mean']})\n",
```

Comment:
this looks similar to your `process_distance_data`, could you reuse that here?

Reply:
The returned data frame has different set of columns in `process_distance_data` than required. We could split the function further, and return the required `dist` data frame. Would you think that would be a better approach?

Reply:
If it's not fairly simple to add it to the function then I think it's fine to keep it as is, it just looked similar at a glance so I was wondering!